packages/block-library/src/html/edit.js

```diff
@@ -17,18 +23,45 @@ import Preview from './preview';
 
 export default function HTMLEdit( { attributes, setAttributes, isSelected } ) {
 	const [ isPreview, setIsPreview ] = useState();
+	const [ shouldFocus, setShouldFocus ] = useState( false );
 	const isDisabled = useContext( Disabled.Context );
 
+	const htmlTextareaRef = useRef();
+	const blockRef = useRef();
+
+	const instanceId = useInstanceId( HTMLEdit );
+	const instanceIdDesc = sprintf( 'html-edit-%d-desc', instanceId );
```

Comment:
```suggestion
	const descriptionId = useInstanceId( HTMLEdit, 'html-edit-desc' );
```

Suggestion: The `useInstanceId` accepts a second `prefix` parameter; it could simplify things.

Reply:
Done.